README.md

```diff
@@ -11,9 +11,81 @@ Federalist is a unified interface for publishing static government websites. It
 
 ## Getting started
 
-To run the server, you'll need [Node.js](https://nodejs.org/download/). You'll also need [nvm](https://github.com/creationix/nvm).
+### Dependencies / Tooling
+Before you start, ensure you have the following installed:
+
+- [node](https://nodejs.org/en/download/package-manager/#osx)
+- [nvm](https://github.com/creationix/nvm#installation) or [n](https://github.com/tj/n#installation)
+- [yarn](https://yarnpkg.com/)
+- [Postgres](https://www.postgresql.org/)
+- [Docker](https://docs.docker.com/engine/installation/mac/)
+- [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+- [AWS CLI](https://aws.amazon.com/cli/)
+
+
+### Running locally (development)
+
+1. Download or clone the `18F/federalist` repository from Github.
+1. Run `nvm use` to ensure you are using the correct version of node
+1. Run `yarn` from the root (the directory that houses the projects files on your computer) of the repository to load modules and install Jekyll dependencies
```

Comment:
remove "from the root (the directory that houses the projects files on your computer) of the repository"

Reply:
(if we add the other suggestion to "`cd` to its directory")